src/test/java/com/ibm/cloud/sdk/core/test/http/HttpConfigTest.java

```diff
@@ -16,13 +22,23 @@
 
   @Test
   public void testHttpConfigOptions() {
+    Authenticator authenticator = new Authenticator() {
+      @Nullable
+      @Override
+      public Request authenticate(@Nullable Route route, Response response) throws IOException {
+        return null;
+      }
+    };
+
     Proxy proxy = new Proxy(Proxy.Type.HTTP, new InetSocketAddress("proxyHost", 8080));
     HttpConfigOptions configOptions = new HttpConfigOptions.Builder()
         .disableSslVerification(true)
         .proxy(proxy)
+            .proxyAuthenticator(authenticator)
```

Comment:
Could you align the indentation here?

Reply:
Wow! What a silly mistake! Sorry. I've fixed it. :)